app/Jobs/SendPostToQuasar.php

```diff
@@ -47,8 +55,10 @@ public function handle()
         }
 
         // Log
-        $verb = $shouldSendToQuasar ? 'sent' : 'would have been sent';
-        info('Post ' . $this->post->id . ' ' . $verb . ' to Quasar');
+        if ($this->log) {
+            $verb = $shouldSendToQuasar ? 'sent' : 'would have been sent';
+            info('Post ' . $this->post->id . ' ' . $verb . ' to Quasar');
```

Comment:
I'm confused - is this still logging every time but just changes the verb?

Reply:
@chloealee great question! The if statement determines if we want to log or not, and the stuff inside is what does the actual logging. When we log, we figure out if we are actually sending it to Quasar or not, and log whether we actually sent it, or that we hit the job and would have sent it if Blink was on. So if Blink is on we log:
Post 4 sent to Quasar
If Blink is off we log:
Post 4 would have been sent to Quasar
Let me know if that's still confusing or doesn't make sense!

Reply:
that makes sense! But wasn't the problem before that we were running into problems because we were logging everything here and we're still logging the same amount? I think I might be misunderstanding something!

Reply:
@chloealee Oh! So in the script where we dispatch the job (https://github.com/DoSomething/rogue/pull/666/files#diff-a29f7802c195289adb5fc1ba1033f452R67) we usually send in an argument of `false` so that `$this->log` will be false and it won't log

Reply:
@katiecrane ohohoh I'm a dummy I was totally reading that wrong - that makes total sense thanks for explaining!

Reply:
haha no it's confusing! thanks @chloealee !